k2/csrc/fsa_utils_test.cu

```diff
@@ -0,0 +1,179 @@
+/**
+ * @brief Unittest for fsa utils.
+ *
+ * @copyright
+ * Copyright (c)  2020  Mobvoi Inc.        (authors: Fangjun Kuang)
+ *
+ * @copyright
+ * See LICENSE for clarification regarding multiple authors
+ */
+
+#include <gtest/gtest.h>
+
+#include "k2/csrc/fsa_utils.h"
+
+namespace k2 {
+
+// clang-format off
+bool operator==(const Arc &a, const Arc &b) {
```

Comment:
I guess the function may be used in other places, so it would be better if we can put it in `Arc`. Anyway, it's OK to leave it here for now.